medmodels/medrecord/tests/test_overview.py

```diff
@@ -0,0 +1,224 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+import polars as pl
+
+import medmodels as mm
+from medmodels.medrecord._overview import extract_attribute_summary, prettify_table
+from medmodels.medrecord.querying import edge, node
+
+
+def create_medrecord():
+    patients = pd.DataFrame(
+        {
+            "index": ["P1", "P2", "P3"],
+            "age": [20, 30, 70],
+        }
+    )
+    diagnosis = pd.DataFrame({"index": ["D1", "D2"]})
+    prescriptions = pd.DataFrame(
+        {"index": ["M1", "M2", "M3"], "ATC": ["B01AF01", "B01AA03", np.nan]}
+    )
+    nodes = [patients, diagnosis, prescriptions]
+    edges = pd.DataFrame(
+        {
+            "source": ["D1", "M1", "D1"],
+            "target": ["P1", "P2", "P3"],
+            "time": ["2000-01-01", "1999-10-15", "1999-12-15"],
+        }
+    )
+    edges.time = pd.to_datetime(edges.time)
+    edges_disease = pl.DataFrame(
+        {
+            "source": ["D1", "D1", "D1"],
+            "target": ["P1", "P2", "P3"],
+            "intensity": [1, "low", None],
+        },
+        strict=False,
+    )
+    groups = [
+        ("Patients", patients["index"].to_list()),
+        ("Stroke", ["D1"]),
+        ("Medications", ["M1", "M2", "M3"]),
+        ("Aspirin", ["M3"]),
+    ]
+    medrecord = mm.MedRecord.from_pandas(
+        nodes=[(node, "index") for node in nodes],
+        edges=(edges, "source", "target"),
+    )
+    medrecord.add_edges_polars(edges=(edges_disease, "source", "target"))
+    for group, group_list in groups:
+        medrecord.add_group(group, group_list)
+
+    return medrecord
```

Comment:
Just FYI: This is les terrible spacing wise

Don't _need_ to fix anything because it's just a test